bentoml/clipper/__init__.py

```diff
@@ -167,12 +167,12 @@ def deploy_bentoml(
             )
         )
 
-    if api_metadata.handler_type not in HANDLER_TYPE_TO_INPUT_TYPE:
+    if api_metadata.input_type not in HANDLER_TYPE_TO_INPUT_TYPE:
         raise BentoMLException(
             "Only BentoService APIs using ClipperHandler can be deployed to Clipper"
         )
 
-    input_type = HANDLER_TYPE_TO_INPUT_TYPE[api_metadata.handler_type]
+    input_type = HANDLER_TYPE_TO_INPUT_TYPE[api_metadata.input_type]
```

Comment:
maybe change this to something like `to_clipper_input_type(api_metadata.input_type` and `CLIPPER_INPUTS`?